### DISCUSSION TITLE
No protein was identified at 1% FDR but report.pg showed proteins

### DISCUSSION THREAD

**Discussion by amakoy124** [vdemichev/DiaNN#529] [Q&A]
Hello,

In one of my search, the log said "Number of proteins identified at 1% FDR: 0 (precursor-level), 0 (protein-level) (inference performed using proteotypic peptides only)", but there were a couple hundreds of proteins (neither contaminants or decoys) listed in the report.pg_matrix.tsv, and I am a bit confused where this discrepancy is coming from. 1% FDR threshold was only applied for precursor ID? I am afraid if this is a naive question, but I'd appreciate any input!

Thank you,
Yuka

**Comment by vdemichev** [comment]
Hi Yuka,

The log refers to protein identifications obtained using exclusively proteotypic peptides, while pg_matrix is based on protein groups.

Best,
Vadim

**Reply by amakoy124** [reply]
Hi Vadim,

Thank you for your prompt response! Proteotypic peptides is equal to unique peptides here?

Thank you,
Yuka

**Reply by vdemichev** [reply]
Yes = peptides specific to the protein according to the 'Protein inference' setting

**Reply by amakoy124** [reply]
Cool. Thank you, Vadim!
